app/src/components/Generation/FloatingGenerateBox.tsx

```
@@ -8,7 +8,9 @@ import { Form, FormControl, FormField, FormItem, FormMessage } from '@/component
 import {
   Select,
   SelectContent,
+  SelectGroup,
   SelectItem,
+  SelectLabel,
```

Biome (2.4.11)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)